api/src/controllers/extensions.ts

```diff
@@ -48,7 +49,8 @@ router.get(
 			throw new ForbiddenError();
 		}
 
-		const { search, limit, offset, type, by, sort } = req.query;
+		const { search, limit, offset, type, sort } = req.query;
+		const { filter } = req.sanitizedQuery as { filter?: { by?: FieldFilterOperator } };
```

Comment:
Let's get all the queries from `sanitizedQuery`:
```suggestion
		const { search, limit, offset, sort, filter } = req.sanitizedQuery;
```